File: go/iq/w-g2~set~group+hidden_group+link+links.go
// BuildGroupsLinkSubGroups builds <iq type="set" xmlns="w:g2">. Replies with one of: Success, ClientError, ServerError.
func BuildGroupsLinkSubGroups(to types.JID, to types.JID, jid types.JID) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"to": to, "xmlns": "w:g2", "type": "set"},
		Content: []waBinary.Node{
			{Tag: "links", Attrs: waBinary.Attrs{}},
		},
	}
}

File: rust/iq/w-g2~set~group+hidden_group+link+links.rs
/// <iq type="set" xmlns="w:g2">. Replies with one of: Success, ClientError, ServerError.
pub fn build_groups_link_sub_groups(to: Jid, to: Jid, jid: Jid) -> Node {
    NodeBuilder::new("iq")
        .attr("to", to)
        .attr("xmlns", "w:g2")
        .attr("type", "set")
        .child(NodeBuilder::new("links").build())
        .build()
}